components/navigation/JobPostItem.js

```diff
@@ -1,46 +1,92 @@
+import { join } from "lodash";
+
 export default function JobPostItem({ job }) {
-  return (    
+  return (
     <li>
       <a href={job.slug} className="block hover:bg-gray-50">
         <div className="px-4 py-4 sm:px-6">
           <div className="flex items-center justify-between">
             <p className="text-sm font-medium text-indigo-600 truncate">
-              { job.title }
+              {job.title}
             </p>
+            <a href={job.company.url}>
```

Comment:
We should not have links inside links. Notice this `<a>` tag is inside another `<a>` tag.

Reply:
for this scenario , someone in slack suggested having a URL of the company website that gives the job.

Reply:
If you still want to have a link to the company then maybe it can be inside the job offer page instead of in the list.